leaf/extensions/tags.py

```diff
@@ -14,6 +14,11 @@ class TagsCog(commands.GroupCog, name="Tags", group_name="tags"):
     def __init__(self, bot: LeafBot) -> None:
         self.bot = bot
 
+    async def check_perms(self, tag_record,  interaction: discord.Interaction) -> bool:
```

Comment:
This should be renamed to check_permissions, add a type hint for tag_record, as well as removing the double space between the comma and the word "interaction".